reliquery/relic.py

```diff
@@ -73,3 +73,24 @@ def get_array(self, name: str) -> np.ndarray:
 
     def list_arrays(self) -> List[np.ndarray]:
         return self.storage.list_keys([self.relic_type, self.name, "arrays"])
+        
+    def add_html(self, name: str, html_path: str):
```

Comment:
you can add return types (similar to the other functions)

Reply:
can do

Reply:
done